qemu/tests/cfg/win_irq_check.cfg

```diff
@@ -84,21 +84,15 @@
                     device_type = "virtio-tablet-pci"
                     input_dev_type_input1 = tablet
     variants:
-        - @default:
         - msi_disable:
```

Comment:
For vioscsi, both @default and msi_disable cases can be deleted, so reset this variants only consider the driver of need this case is ok.

Thanks~
Peixiu